protofsm/state_machine.go

```diff
@@ -197,7 +200,10 @@ func NewStateMachine[Event any, Env Environment](
 	cfg StateMachineCfg[Event, Env]) StateMachine[Event, Env] {
 
 	return StateMachine[Event, Env]{
-		cfg:            cfg,
+		cfg: cfg,
+		log: log.WithPrefix(
+			fmt.Sprintf("FSM(%v)", cfg.Env.Name()),
```

Comment:
nit: should we add a `:` or a space?

Reply:
cool yeah added `:` (the space is added by the underlying logging library)